pkg/proto/datadog/model/model.proto

```diff
@@ -0,0 +1,100 @@
+syntax = "proto3";
```

Comment:
`api/v1/api.proto` is versioned and depends on this model so it would make sense that this proto is versioned too otherwise they are likely to fall out of sync.